syntax = "proto3";

option csharp_namespace = "ChessCom";

package chesscom;

// Defines the service
service ChessCom {
    // Function invoked to send the request
    rpc sendRequest (MathRequest) returns (MathReply) {}
}

// The request message containing requested numbers
message MathRequest {
    int32 a = 1;
    int32 b = 2;
}

// The response message containing response
message MathReply {
    int32 result = 1;
}
